content/tutorials/token-authentication.textile

```diff
@@ -34,13 +34,9 @@ tags:
     - Authentication
 ---
 
-Ably supports two types of authentication schemes. Basic authentication uses one of your private API keys and is the simplest scheme designed for use by your servers. Token authentication is mostly used by your device and browser clients whereby a short-lived secure token is issued to them by your servers.
+Ably supports two types of authentication schemes, basic authentication using a private API key, which should not be used on user devices, and "token authentication":/core-features/authentication#token-authentication which can be used on your server to keep your private API key secure. Your server (or serverless function) can generate short-lived tokens and pass them to users when they need to communicate with Ably. These tokens can be limited to specific capabilities on specific channels. Make sure to follow the principle of least privilege, giving clients only the capabilities they need.  
```

Comment:
Can we link to the [basic authentication scheme docs](https://ably.com/documentation/core-features/authentication#basic-authentication) here too please? 

Also, rather than talking about "used on user devices" we could perhaps state that because basic authentication repeatedly sends the credentials on each request it is not the most secure method of authentication and therefore shouldn't be considered in production applications, where the private API key should only ever be stored server-side.

Reply:
Fixed in 5766dba